src/redux/categories/categoriesSlice.js

```diff
@@ -0,0 +1,13 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+const initialState = {
+  categories: [{ status: 'Under construction' }],
+};
+
+const categoriesSlice = createSlice({
+  name: 'categories',
+  initialState,
+  reducers: {},
+});
+
+export default categoriesSlice.reducer;
```

Comment:
- [ ] Let's improve the construction of the _CategoriesSlice_ by ensuring the following:
   - Creates an initialState with a state value categories that contains an empty array
   - Creates 1 reducer, checkStatus
   - Exports by checkStatus as an action

And then the CategriesSlice should be similar to:

```javascript
// ./src/redux/categories/categoriesSlice.js

import { createSlice } from "@reduxjs/toolkit"

export const categoriesSlice = createSlice({
  name: "categories",
  initialState: {
    categories: [],
  },
  reducers: {
    checkStatus: (state, action) => {
      state.categories =
        action.payload === "Under construction"
          ? "Under construction"
          : state.categories
    },
  },
})

export const { checkStatus } = categoriesSlice.actions

export default categoriesSlice.reducer
```

Reply:
Thanks for the good insight 😃